config/bartworks.cfg

```diff
@@ -1,446 +1,501 @@
-
-    # If GTNH is Loaded, this will enable easy recipes, if not, it will enable harder recipes.
-    B:"Mode switch"=false
+general {
+
+    ##########################################################################################################
+    # mixins
+    #--------------------------------------------------------------------------------------------------------#
+    # Mixins section.
+    ##########################################################################################################
+
+    mixins {
+        # if true, patches the crafting manager to cache recipes in class: net.minecraft.item.crafting.CraftingManager [default: false]
+        B:enableCraftingManagerRecipeCaching=false
+    }
+
+    ##########################################################################################################
+    # crossmodinteractions
+    #--------------------------------------------------------------------------------------------------------#
+    # Crossmod interactions section.
+    ##########################################################################################################
+
+    crossmodinteractions {
+        # If the Ross128 System should be activated, DO NOT CHANGE AFTER WORLD GENERATION [default: true]
+        B:Ross128Enabled=true
+
+        # True disables the magical Forest Biome on Ross for more performance during World generation. [default: false]
+        B:disableMagicalForest=false
+
+        # 1 = Moon Lander, 2 = Landing Balloons, 3 = Asteroid Lander
+        # Possible values: [MoonLander, LandingBalloons, AsteroidsLander]
+        #  [default: AsteroidsLander]
+        S:landerType=AsteroidsLander
+
+        # The Dim ID for Ross128ba (Ross128b's Moon) [range: -2147483648 ~ 2147483647, default: 63]
+        I:ross128BAID=63
+
+        # The Dim ID for Ross128b [range: -2147483648 ~ 2147483647, default: 64]
+        I:ross128BID=64
+
+        # Higher Values mean lesser Ruins. [range: -2147483648 ~ 2147483647, default: 512]
+        I:ross128bRuinChance=512
+    }
+
+    ##########################################################################################################
+    # multiblocks
+    #--------------------------------------------------------------------------------------------------------#
+    # Multiblocks section.
+    ##########################################################################################################
+
+    multiblocks {
+        # This switch enables the Direct Steam Mode of the DEHP. If enabled it will take in Waterand output steam. If disabled it will Input IC2Coolant and output hot coolant [default: false]
+        B:DEHPDirectSteam=false
+
+        # This switch completely disables piston animation in Electric Implosion Compressor multiblock [default: false]
+        B:disablePistonInEIC=false
+
+        # This will set Up the Energy per LESU Cell [range: -2147483648 ~ 2147483647, default: 1000000]
+        I:energyPerCell=1000000
```

Comment:
not like lesu is important but it looks like you nerfed this

Reply:
back to 20M